lib/fastlane/fast_file.rb

```diff
@@ -206,19 +206,13 @@ def import_from_git(url: nil, branch: 'HEAD', path: 'fastlane/Fastfile')
         # Checkout the repo
         repo_name = url.split("/").last
 
-        clone_folder = File.join("/tmp", "fl_clones", repo_name)
+        clone_folder = File.join("/tmp", "fl_clones_#{Time.now.to_f}", repo_name)
 
         branch_option = ""
         branch_option = "--branch #{branch}" if branch != 'HEAD'
 
         clone_command = "git clone '#{url}' '#{clone_folder}' --depth 1 -n #{branch_option}"
 
-        if Dir.exist? clone_folder
```

Comment:
Why wouldn't we still want to delete the old folder? Otherwise we're spamming the `/tmp` folder so much.